controller/src/main/java/com/softserve/borda/controllers/BoardController.java

```diff
@@ -154,11 +154,11 @@ public Board getBoard(@PathVariable Long id) {
     public ResponseEntity<BoardListDTO> createBoardListsForBoard(@PathVariable Long boardId,
                                                                  @RequestBody BoardListDTO boardListDTO) {
         try {
-            BoardList boardList = new BoardList();
-            boardList.setName(boardListDTO.getName());
-            boardList = boardListService.create(boardList);
-            boardList = boardListService.addBoardListToBoard(boardId, boardList.getId());
-            boardListDTO = modelMapper.map(boardList, BoardListDTO.class);
+            BoardColumn boardColumn = new BoardColumn();
+            boardColumn.setName(boardListDTO.getName());
+            boardColumn = boardColumnService.create(boardColumn);
+            boardColumn = boardColumnService.addBoardListToBoard(boardId, boardColumn.getId());
```

Comment:
Does BoardColumn entity (or table) contain reference on Board entity?

Reply:
Not yet. Will be fixed in a separate branch later.

Reply:
Separate ticket has been assigned for this issue: https://trello.com/c/or4iVPK2/152-re-check-entity-mappings-and-fetching